autoload/neomake/makers/ft/python.vim

```diff
@@ -152,11 +143,12 @@ function! neomake#makers#ft#python#pep257() abort
 endfunction
 
 function! neomake#makers#ft#python#PylamaEntryProcess(entry) abort
+    echom a:entry.text
     if a:entry.type ==# 'C' && a:entry.text =~# '\v\[%(pycodestyle|pep8)\]$'
         call neomake#makers#ft#python#Pep8EntryProcess(a:entry)
     elseif a:entry.type ==# 'D'  " pydocstyle/pep257
         let a:entry.type = 'W'
-    elseif a:entry.type ==# 'E901'  " mccabe
+    elseif a:entry.text =~# '^C901'  " mccabe
```

Comment:
What about `type ==# 'C' && `nr == 901`?  (but feel free to keep it like it is now).